source/index.md

```diff
@@ -600,9 +602,50 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
-##### Method: `mobilepayonline`
 
-<dl class="dl-horizontal">
+---
+##### **Method**: `googlepay`
+
+To accept a payment using Google Pay, apart from the complete payment method
+token and merchant ID, the derived shared secret is required.
+Please refer to the [official documentation][GooglePay-PaymentCryptography].
+Supported protocol versions are `ECv1` and `ECv2`.
+
+<dl>
+  <dt>googlepay[token]</dt>
+  <dd>[:json:]<br />Raw payment method token as received in response from Google. UTF-8
+    encoded serialization of a JSON dictionary.</dd>
+  <dt>googlepay[shared_secret]</dt>
```

Comment:
Minor detail, but Google is calling this a "shared key".

Reply:
@kse-clearhaus shouldn't we just update the API (and the implementation) accordingly? Or was there a reason for the current naming?

Reply:
```suggestion
  <dt>googlepay[shared_key]</dt>
```

Reply:
Before committing the suggestion, please notice https://github.com/clearhaus/gateway-api-docs/pull/69#issuecomment-578094920 🙂

Reply:
Yeah, they likely changed it from version 1 to 2. I remember stressing about the names.

Should we change it? Hmm.

Reply:
I think we should basically just use whatever Google names it to require as little translation as possible. It's really to cheap to change it now, so I'd say that we should just change it right away; I'll commit it in this PR, and we can simply adjust this PR back if necessary. Thereby, I'll resolve this conversation.